lib/routes/polls.js

```diff
@@ -0,0 +1,39 @@
+const { Router } = require('express');
+const Poll = require('../models/Poll');
+
+module.exports = Router()
+  .post('/', (req, res, next) => {
+    Poll
+      .create(req.body)
+      .then(poll => res.send(poll))
+      .catch(next);
+  })
+  .get('/', (req, res, next) => {
+    Poll
+      .find(req.query)
+      .select({ 
+        _id: true,
+        title: true
+      })
+      .then(polls => res.send(polls))
+      .catch(next);
+  })
+  .get('/:id', (req, res, next) => {
+    Poll
+      .findById(req.params.id)
+      .populate('organization', { id:true, title: true })
```

Comment:
also need to populate your vote count here https://mongoosejs.com/docs/populate.html#count